packages/application-extension/src/index.tsx

```diff
@@ -327,6 +330,16 @@ const mainCommands: JupyterFrontEndPlugin<void> = {
         isToggled: () => !labShell.rightCollapsed,
         isEnabled: () => !labShell.isEmpty('right')
       });
+      //WIP
+      commands.addCommand(CommandIDs.focusFirstElementInRightSidebar, {
+        label: trans.__('Focus Right Sidebar First Element'),
+        execute: () => {
+          let rightSidBarFirstElement = document.getElementById('tab-key-1-1');
```

Comment:
The element should be retrieved programatically from `labShell` - relying from on element IDs is error-prone because we would not notice that something is wrong if the IDs were changed. IDs would be re acceptable if there is _really_ no other way and there is an accompanying test case.